### PR TITLE
UIROLES-86: Use `Capabilities` components from `stripes-authorization-components` repository instead of local files

### PR DESCRIPTION
**Pull request by alisher-epam**
Refs [UIROLES-86](https://folio-org.atlassian.net/browse/UIROLES-86)

Extract `Capabilities` related components to the new share repository and import Components from the repo https://github.com/folio-org/stripes-authorization-components

### CODE REVIEW

**Comment by alisher-epam** [comment]
Once the PR is merged all the test cases pass correctly. https://github.com/folio-org/stripes-authorization-components/pull/2

If you would like to check the tests you can add this config to the `package.json` file

```
"resolutions": {
    "@folio/stripes-authorization-components": "https://github.com/folio-org/stripes-authorization-components.git#UISAUTHCOM-1"
  }
  ```

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## 1.6.0 IN PROGRESS
 
 * Show modifying users' names instead of IDs. Refs UIROLES-71.
+* Use `Capabilities` components from `stripes-authorization-components` repository instead of local files. Refs UIROLES-86.
```
Comment (BogdanDenis) [inline CHANGELOG.md:6]: Please mark it as a breaking change, since a new dependency in introduced

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (NikitaSedyx) [reply CHANGELOG.md]: new dependency doesn't mean it's breaking change

Comment (BogdanDenis) [reply CHANGELOG.md]: @NikitaSedyx you're right, I thought it was a new peerDep

File: src/settings/SettingsPage/SettingsPage.test.js [hunk 2]
```diff
@@ -1,34 +1,25 @@
-import React from 'react';
 import userEvent from '@folio/jest-config-stripes/testing-library/user-event';
 import { render } from '@folio/jest-config-stripes/testing-library/react';
+import { useRoleCapabilities, useAuthorizationRoles } from '@folio/stripes-authorization-components';
```
Comment (usavkov-epam) [inline src/settings/SettingsPage/SettingsPage.test.js:3]:
```suggestion
import {
  useRoleCapabilities,
  useAuthorizationRoles,
} from '@folio/stripes-authorization-components';
```

**Review by usavkov-epam — APPROVED** [review]
Before merging, please make sure one of the approvals belongs to developers from the Eureka team.

**Comment by alisher-epam** [comment]
@BogdanDenis please review it

File: CHANGELOG.md (hunk 1 above)
Comment (alisher-epam) [reply CHANGELOG.md:6]: @BogdanDenis please approve it

**Review by BogdanDenis — APPROVED** [review]

**Pull request merged**